src/solc_0.7/common/Libraries/AddressUtils.sol

```diff
@@ -0,0 +1,20 @@
+//SPDX-License-Identifier: MIT
+pragma solidity 0.7.1;
+
+library AddressUtils {
+    function toPayable(address _address) internal pure returns (address payable _payable) {
+        return address(uint160(_address));
+    }
+
+    function isContract(address addr) internal view returns (bool) {
```

Comment:
we should use openzeppelin version

Reply:
Agreed. I think we already switched it in ERC721BAseToken.sol. I'll look at removing our custom implementation.

Reply:
removed.